controllers/organization_validator.go

```diff
@@ -74,19 +72,12 @@ func (v *OrganizationValidator) handle(ctx context.Context, req admission.Reques
 		}
 	}
 
-	orgClusters := &unstructured.UnstructuredList{}
-	orgClusters.SetGroupVersionKind(schema.GroupVersionKind{
-		Group:   "cluster.x-k8s.io",
-		Version: "v1alpha2",
-		Kind:    "cluster",
-	})
+	orgClusters := capi.ClusterList{}
 	{
-		err := v.List(ctx, orgClusters, &client.ListOptions{
+		err := v.List(ctx, &orgClusters, &client.ListOptions{
 			LabelSelector: orgClustersSelector,
 		})
-		if apimeta.IsNoMatchError(err) {
```

Comment:
What about the case where the CRD isn't ensured? On KVM we don't have that CRD, and deletion should succeed without the cluster check

Reply:
Oh I see. Good catch! Let me push again.